aiogram/api/types/base.py

```diff
@@ -5,7 +5,7 @@
 from aiogram.utils.mixins import ContextInstanceMixin
 
 
-class TelegramObject(ContextInstanceMixin, BaseModel):
+class TelegramObject(ContextInstanceMixin["TelegramObject"], BaseModel):
```

Comment:
In due to [PEP-563](https://www.python.org/dev/peps/pep-0563/) you can remove here brackets in type hint by adding in top of file next line:
```python3
from future import __annotations__
```

And then you can do this:
```suggestion
class TelegramObject(ContextInstanceMixin[TelegramObject], BaseModel):
```